src/Symfony/Component/Cache/Tests/Adapter/AdapterTestCase.php

```diff
@@ -52,8 +52,6 @@ public function testGet()
 
         $this->assertNull($cache->get('foo', function (CacheItem $item) use (&$isHit, $value) {
             $isHit = false;
-            $this->assertTrue($item->isHit());
-            $this->assertSame($value, $item->get());
```

Comment:
I suppose this is removed to make `NullAdapter` pass this test?
what about skipping this test case for it specifically instead, using the skippedTests property?

Reply:
Actually, nope, it's for the ArrayAdapter to pass this test, NullAdapter own tests were fine: it's linked to my question in the issue #27731 - since when rebuild conditions are met before loading the item, I just do not load it, hence the $newItem and those lines failing. Let's continue this discussion in #27731.